sql/core/src/test/resources/sql-tests/results/date_part.sql.out

```diff
@@ -857,7 +857,7 @@ select date_part('epoch', c) from t2
 -- !query 106 schema
 struct<date_part('epoch', t2.`c`):decimal(18,6)>
 -- !query 106 output
-31873892788.332003
+31897220765.004003
```

Comment:
Thanks, @yaooqinn . This looks enough. Shall we revert the changes on `IntervalUtilsSuite.scala`?

Reply:
OK